HISTORY.rst

```diff
@@ -10,6 +10,7 @@ Unreleased Changes
   called with an invalid resampling algorithm. We now fall back to the
   underlying GDAL functions' error messages.
   https://github.com/natcap/pygeoprocessing/issues/387
+* Updated to Cython 3.
```

Comment:
If we are removing testing/wheel-building support for python 3.7, should that also be noted here in HISTORY and also removed from the `setup.py` trove classifiers?

Reply:
Ooh you're right